vale/dictionaries/en_US-grafana.dic

```diff
@@ -36,6 +36,8 @@ JMeter/ po:noun
 Jsonnet/ po:noun
 JUnit/ po:noun
 Kinesis/ po:noun
+KPI/ po:noun
+KPIs/ po:noun
```

Comment:
This is just a nit of how the dictionary works but with a `/S` we can have a single entry for this work plus the pluralization which comes from the affix rule in https://github.com/grafana/writers-toolkit/blob/main/vale/dictionaries/en_US-grafana.aff#L20-L24.

```suggestion
KPI/S po:noun
```

Reply:
@jdbaldry oh cool, thanks!

Reply:
I've already applied the suggestion but I've left the comment unresolved as explanation.